pavelib/paver_tests/utils.py

```diff
@@ -4,6 +4,11 @@
 from paver import tasks
 from unittest import TestCase
 
+try:
+    unicode        # Python 2
+except NameError:  # Python 3
+    unicode = str  # pylint: disable=invalid-name,redefined-builtin
+
```

Comment:
Per [OEP-7](https://open-edx-proposals.readthedocs.io/en/latest/oep-0007-bp-migrate-to-python3.html), we would rather use six than use ad-hoc compatibility code like this.

Reply:
Done.